packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@calyptia-vivo/components",
-  "version": "0.0.4",
+  "version": "0.0.9-2",
```

Comment:
Because it is a pull request, so it advances --prerelease
https://classic.yarnpkg.com/lang/en/docs/cli/version/

A merge to master will advance --patch

Reply:
@dfredriksen okay, but even so - to go from `0.0.4` to `0.0.9-2` because this increments on every commit doesn't make sense. I also don't think we want "dash" versioning numbers. Happy to discuss.

Reply:
We can put a prefix, such as "-pr0.1" but the "prelease" part goes away once "patch" is incremented on merge to main. We can put a parking lot item in place to discuss today

Reply:
1093799b32b884ce815f4b244447564d6cca27c6 only will publish the package if the components path is changed.